test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -818,6 +818,11 @@ Verb-Noun -Param1 Hello ^
         $res.CompletionMatches[0].CompletionText | Should -Be "Break"
     }
 
+    it 'Should complete command with an empty arrayexpression element' {
+        $res = TabExpansion2 -inputScript 'Get-ChildItem @()' -cursorColumn 1
```

Comment:
Why not use `Get-Chi<tab> @()` as is called out in the PR description?

```suggestion
        $res = TabExpansion2 -inputScript 'Get-Chi @()' -cursorColumn 7
```

Reply:
To make it less likely for the test to break if a new command gets added.

Reply:
I see. Then we are good.